handlers/handler-loadbalance/src/main/java/org/apache/servicecomb/loadbalance/RuleNameExtentionsFactory.java

```diff
@@ -51,13 +47,13 @@ public boolean isSupport(String key, String value) {
   }
 
   @Override
-  public IRule createLoadBalancerRule(String ruleName) {
+  public RuleExt createLoadBalancerRule(String ruleName) {
     if (RULE_RoundRobin.equals(ruleName)) {
```

Comment:
support SPI?
and use a map to manage them?

Reply:
Now the extension point is ExtensionsFactory, not RuleExt.